narwhals/_dask/namespace.py

```diff
@@ -50,6 +50,18 @@ def col(self, *column_names: str) -> DaskExpr:
             backend_version=self._backend_version,
         )
 
+    def all(self) -> DaskExpr:
+        return DaskExpr(
+            lambda df: [
+                df._native_dataframe.loc[:, column_name] for column_name in df.columns
+            ],
```

Comment:
I feel somehow a bit uncomfy returning a native object, but since we don't have a DaskSeries I think that's ok?!

Reply:
yup!